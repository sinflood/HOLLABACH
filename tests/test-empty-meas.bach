inst Banjo {
[A2/4]
[]
[A2/4]
}
