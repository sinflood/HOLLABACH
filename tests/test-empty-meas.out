Timing Resolution (pulses per quarter note)
4

Instrument,105,,

Tick,Note(0-127),Velocity(0-127),
0,45,1
32,45,1
